services/app-web/src/components/SubmissionSummarySection/ContractDetailsSummarySection/ContractDetailsSummarySection.test.tsx

```diff
@@ -235,15 +235,17 @@ describe('ContractDetailsSummarySection', () => {
 
     it('does not render download all button when on previous submission', () => {
         renderWithProviders(
-            <Route
-                path={RoutesRecord.SUBMISSIONS_REVISION}
-                component={() => (
-                    <ContractDetailsSummarySection
-                        submission={draftContractAndRatesSubmission}
-                        submissionName="MN-PMAP-0001"
-                    />
-                )}
-            />,
+            <Routes>
+                <Route
+                    path={RoutesRecord.SUBMISSIONS_REVISION}
```

Comment:
It would be the same in this instance good catch! There are some tests with the same pattern that does require `Routes` and `Route` for components that calls certain hooks from `react-router` like `useParams`. There is probably a way to integrate using these hooks into render options, but what do you think about spinning that off to a different task?